doc/parameters.md

```diff
@@ -241,6 +242,8 @@ Replace `${ID}` with the actual Service ID. The value should be the configuratio
 
 Setting it to a particular version will prevent it from auto-updating and will always use that version.
 
+**Note**: This env var cannot be used with `THREESCALE_PORTAL_ENDPOINT` pointing to custom path (i.e. master path).
```

Comment:
Does that mean this variable won't work with basic staging/production APIcast created by 3scale operator? Did it work before and did it changed now or has it simply never worked before?

Reply:
Never worked before. The env var was silently ignored. Now (actually in the previous PR), I added a hard [check](https://github.com/3scale/APIcast/pull/1397/files#diff-45d2061c12ed47f3c6c589f0aa51e73640f5518ee3b97085a85d9822a9c9618fR340). The check will not kill the process, but will not process the request and will log error.